### PR TITLE
chore: convert project to monorepo and add coordinator service

### PR DESCRIPTION
**Pull request by ctrlc03**
Covert project to a monorepo, add coordinator service.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
 .eslintrc.js
 commitlint.config.js
-subgraph/generated
-public/mockServiceWorker.js
 zkeys
-playwright-report
-test-results
-next.config.js
+interface/playwright.config.ts 
+interface/playwright/fixtures.ts
```
Comment (0xmad) [inline .eslintignore:10]: Let's add new line at the end

File: .husky/pre-commit [hunk 2]
```diff
@@ -1 +1 @@
-pnpm exec lint-staged && pnpm run types
+pnpm lint && pnpm run types
```
Comment (0xmad) [inline .husky/pre-commit:1]: lint-staged is not the same as lint command. It allows to run linter only for changed files so let's keep this for pre commit.

File: package.json [hunk 3]
```diff
@@ -1,159 +1,64 @@
...
-    "tailwind-variants": "^0.1.20",
-    "viem": "^2.7.15",
-    "wagmi": "^2.9.8",
-    "zod": "3.22.4"
+    "lint": "pnpm run lint:coordinator && pnpm run lint:interface",
+    "lint:coordinator": "eslint './packages/coordinator/**/*.ts'",
+    "lint:coordinator:fix": "eslint './packages/coordinator/**/*.ts' --fix",
+    "lint:interface": "lerna run lint --scope=maci-platform-interface",
+    "lint:interface:fix": "lerna run lint:fix --scope=maci-platform-interface",
+    "lint:fix": "pnpm run lint:coordinator:fix && pnpm run lint:interface:fix",
```
Comment (0xmad) [inline package.json:20]: It's better to do it like it's implemented for maci repo so we don't need to create command in root project to run linter/prettier and etc.

**Review by 0xmad — CHANGES_REQUESTED** [review]
@ctrlc03 thanks, just left some comments

File: .eslintignore [hunk 4]
```diff
@@ -1,14 +1,10 @@
...
 .eslintrc.js
 commitlint.config.js
-subgraph/generated
-public/mockServiceWorker.js
 zkeys
-playwright-report
-test-results
-next.config.js
+packages/interface/playwright/fixtures.ts
+packages/interface/playwright.config.ts
```
Comment (0xmad) [inline .eslintignore:10]: There is still no new line at the end

Comment (ctrlc03) [reply .eslintignore]: my bad, should be in now

**Review by 0xmad — APPROVED** [review]
@ctrlc03 thanks!

**Pull request merged**